code/modules/clothing/masks/stone.dm

```diff
@@ -82,3 +74,20 @@
 
 /obj/item/clothing/mask/stone/infinite //this mask can be used any number of times
 	infinite = 1
+
+/proc/makeLateVampire(var/mob/living/carbon/human/H, var/blood_to_give)
+	var/datum/role/vampire/vamp =  new(H.mind, override = TRUE)
```

Comment:
There's an argument in new role for assigning a faction. Create the faction, then have it in the constructor arguments of the role. Unless the vampire factions are so bass ackwards as to go against this design.